src/arch/x86/isa.hh

```diff
@@ -93,6 +94,8 @@ class ISA : public BaseISA
     void setThreadContext(ThreadContext *_tc) override;
 
     std::string getVendorString() const;
+
+    X86CPUID *cpuid;
```

Comment:
Probably should be `unique_ptr`

Reply:
I would like to see this change before merging.

Reply:
Any reason not to make this a `unique_ptr`?

Reply:
Missed this conversation somehow (kind find them hard to see in github's dark mode).  Done now.